### DISCUSSION TITLE
Genau das, was ich gesucht habe! Muss aber den Tibber Abruf anpassen.

### DISCUSSION THREAD

**Discussion by Martin-ausO** [camueller/node-red-contrib-heatpump-tibber-pv#2] [Allgemein]
Hallo,
Genau dieses Thema habe ich gesucht!
Kann aber keinen Vertrag mit Tibber machen, weil mein Smart Meter nicht mit dem Puls zusammen arbeitet.
Daher muss ich die Preisabfrage des Börsenstrompreises umstellen.
Versuche es mit Awattar:

https://api.awattar.de/v1/marketdata
das geht ohne Anmeldung.
Brauche aber Unterstützung beim Anpassen der Auswertefunktion, da ich die vorhandene nicht richtig verstehe.

**Comment by camueller** [comment]
Dank OpenSource kannst Du das natürich für Awattar anpassen, aber dafür kann ich keine Unterstützung bieten.

**Comment by Martin-ausO** [comment]
Habe ich vermutet. Dann muss ich mich wohl in die Tiefen der JSON Objekte reinfummeln. Ich vermute, eine Übersetzungsfuntion dürfte das lösen.

**Comment by camueller** [comment]
Ich habe mal einen Blick auf die Awattar-Daten geworfen. Die Daten sehen ähnlich aus wie bei Tibber (wie auch sonst ...). Insofern musst Du nur den Node "hourly energy price" anpassen. Falls Du es nicht schaffst, kann ich Dir auch über den [kostenplichtigen Support](https://github.com/camueller/node-red-contrib-heatpump-tibber-pv/blob/main/Support.md) helfen.

**Comment by Martin-ausO** [comment]
Noch eine Frage. Wo gebe ich die Variablen ein? Muss ich das beim Import in den entsprechenden Feldern tun? Ich finde in Nede Red die entsprechenden Felder nach dem Import nicht wieder.

**Reply by Martin-ausO** [reply]
Der im Video zu sehende Config Node ist bei mir nach Import des Scriptes nicht vorhanden!

**Reply by Martin-ausO** [reply]
![IMG_1685](https://github.com/camueller/node-red-contrib-heatpump-tibber-pv/assets/172635299/b3f9b762-51f1-485e-80ec-a31fb3af4fed)

**Comment by camueller** [comment]
Es gibt jetzt ein Folge-Projekt, dass diverse Anbieter für Strompreise (inkl. Awattar) und PV-Prognosen unterstützt.
https://github.com/camueller/node-red-contrib-heatpump-dynamic-tariff
